journeys/et/dockerCommon.ts

```diff
@@ -79,6 +80,31 @@ export async function configsJourney() {
   }
 }
 
+async function waitForDmStore() {
+  let timeout = 30
+  let left = timeout
+  while (true) {
```

Comment:
check ready outside of loop and then while (!ready). It avoids the while(true) and first guard clause.